### ISSUE TITLE
Leftover merge conflicts

### ISSUE THREAD

**Issue by hlinnaka** [greenplum-db/gpdb#10915]
As @ggbq noted at https://github.com/greenplum-db/gpdb/pull/10883#issuecomment-698284970, there are a bunch of merge conflicts left in the tree after the v12 merge:
```
postgres=# \h COPY
Command:     COPY
Description: copy data between a file and a table
Syntax:
COPY table_name [ ( column_name [, ...] ) ]
    FROM { 'filename' | PROGRAM 'command' | STDIN }
    [ [ WITH ] ( option [, ...] ) ]
<<<<<<< HEAD
    [ ON SEGMENT ]
    [ FILL MISSING FIELDS ]
    [ [LOG ERRORS] SEGMENT REJECT LIMIT count [ [ROWS] | PERCENT ] ]
=======
    [ WHERE condition ]
>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196

COPY { table_name [ ( column_name [, ...] ) ] | ( query ) }
    TO { 'filename' | PROGRAM 'command' | STDOUT }
    [ [ WITH ] ( option [, ...] ) ]
    [ ON SEGMENT ]
    [ IGNORE EXTERNAL PARTITIONS ]

where option can be one of:

<<<<<<< HEAD
    FORMAT 'format_name'
    OIDS [ boolean ]
=======
    FORMAT format_name
>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196
    FREEZE [ boolean ]
    DELIMITER 'delimiter_character'
    NULL 'null_string'
    HEADER [ boolean ]
    QUOTE 'quote_character'
    ESCAPE 'escape_character'
    NEWLINE 'newline_character'
    FORCE_QUOTE { ( column_name [, ...] ) | * }
    FORCE_NOT_NULL ( column_name [, ...] )
    FORCE_NULL ( column_name [, ...] )
    ENCODING 'encoding_name'

URL: https://www.postgresql.org/docs/12/sql-copy.html

contrib/test_decoding/expected/oldest_xmin.out
contrib/test_decoding/expected/rewrite.out
doc/src/sgml/ref/alter_role.sgml
doc/src/sgml/ref/alter_table.sgml
doc/src/sgml/ref/alter_type.sgml
doc/src/sgml/ref/alter_user_mapping.sgml
doc/src/sgml/ref/comment.sgml
doc/src/sgml/ref/copy.sgml
doc/src/sgml/ref/create_aggregate.sgml
doc/src/sgml/ref/create_foreign_data_wrapper.sgml
doc/src/sgml/ref/create_foreign_table.sgml
doc/src/sgml/ref/create_function.sgml
doc/src/sgml/ref/create_index.sgml
doc/src/sgml/ref/create_role.sgml
doc/src/sgml/ref/create_server.sgml
doc/src/sgml/ref/create_table.sgml
doc/src/sgml/ref/create_trigger.sgml
doc/src/sgml/ref/create_user.sgml
doc/src/sgml/ref/create_view.sgml
doc/src/sgml/ref/grant.sgml
doc/src/sgml/ref/initdb.sgml
doc/src/sgml/ref/pg_basebackup.sgml
doc/src/sgml/ref/pg_dump.sgml
doc/src/sgml/ref/psql-ref.sgml
doc/src/sgml/ref/select.sgml
src/pl/plpython/expected/plpython_types_3.out
src/test/isolation/expected/eval-plan-qual.out
src/test/isolation/specs/eval-plan-qual.spec
```

**Comment by ashwinstar** [comment]
Just for completeness recording the list here
```
git grep -w "<<<<<<< HEAD"
contrib/test_decoding/expected/oldest_xmin.out:<<<<<<< HEAD
contrib/test_decoding/expected/oldest_xmin.out:<<<<<<< HEAD
contrib/test_decoding/expected/rewrite.out:<<<<<<< HEAD
doc/src/sgml/ref/alter_role.sgml:<<<<<<< HEAD
doc/src/sgml/ref/alter_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/alter_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/alter_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/alter_type.sgml:<<<<<<< HEAD
doc/src/sgml/ref/alter_user_mapping.sgml:<<<<<<< HEAD
doc/src/sgml/ref/comment.sgml:<<<<<<< HEAD
doc/src/sgml/ref/copy.sgml:<<<<<<< HEAD
doc/src/sgml/ref/copy.sgml:<<<<<<< HEAD
doc/src/sgml/ref/copy.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_aggregate.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_aggregate.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_foreign_data_wrapper.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_foreign_data_wrapper.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_foreign_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_foreign_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_function.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_index.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_role.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_role.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_server.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_server.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_trigger.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_trigger.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_user.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_view.sgml:<<<<<<< HEAD
doc/src/sgml/ref/create_view.sgml:<<<<<<< HEAD
doc/src/sgml/ref/grant.sgml:<<<<<<< HEAD
doc/src/sgml/ref/grant.sgml:<<<<<<< HEAD
doc/src/sgml/ref/grant.sgml:<<<<<<< HEAD
doc/src/sgml/ref/grant.sgml:<<<<<<< HEAD
doc/src/sgml/ref/initdb.sgml:<<<<<<< HEAD
doc/src/sgml/ref/pg_basebackup.sgml:<<<<<<< HEAD
doc/src/sgml/ref/pg_basebackup.sgml:<<<<<<< HEAD
doc/src/sgml/ref/pg_basebackup.sgml:<<<<<<< HEAD
doc/src/sgml/ref/pg_dump.sgml:<<<<<<< HEAD
doc/src/sgml/ref/pg_dump.sgml:<<<<<<< HEAD
doc/src/sgml/ref/pg_dump.sgml:<<<<<<< HEAD
doc/src/sgml/ref/psql-ref.sgml:<<<<<<< HEAD
doc/src/sgml/ref/psql-ref.sgml:<<<<<<< HEAD
doc/src/sgml/ref/select.sgml:<<<<<<< HEAD
src/test/isolation/expected/eval-plan-qual.out:<<<<<<< HEAD
src/test/isolation/expected/eval-plan-qual.out:<<<<<<< HEAD
src/test/isolation/specs/eval-plan-qual.spec:<<<<<<< HEAD
src/test/isolation/specs/eval-plan-qual.spec:<<<<<<< HEAD
src/test/isolation/specs/eval-plan-qual.spec:<<<<<<< HEAD
src/test/isolation/specs/eval-plan-qual.spec:<<<<<<< HEAD
```

**Comment by kainwen** [comment]
I run `git grep -w <<<<<<< HEAD`

```
gpadmin@zlyu:~/workspace/gpdb$ git grep -w "<<<<<<< HEAD"
doc/src/sgml/ref/create_table.sgml:<<<<<<< HEAD
doc/src/sgml/ref/psql-ref.sgml:<<<<<<< HEAD
doc/src/sgml/ref/psql-ref.sgml:<<<<<<< HEAD
```

Most of conflicts are fixed by @ashwinstar 's commit: https://github.com/greenplum-db/gpdb/commit/675ac5835499fc76b3acc67b5e63d7cb87fbb306

**Comment by adam8157** [comment]
Fixed by #12577, none left.
```
$ git grep -w '<<<<<<< HEAD'|wc -l
       0
```
